fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -499,9 +499,10 @@ public class Config extends ConfigBase {
      * The connection timeout and socket timeout config for thrift server
      * The value for thrift_client_timeout_ms is set to be larger than zero to prevent
      * some hang up problems in java.net.SocketInputStream.socketRead0
+     * default is 8h
      */
     @ConfField
-    public static int thrift_client_timeout_ms = 0;
+    public static int thrift_client_timeout_ms = 28800000;
```

Comment:
Why not be equal to the query time for insert?

Reply:
This is the timeout for inactive connection. The query timeout is the process timeout of query, and the task send rpc and task report rpc can use different connection. And the rpc process is very short time. How can these two times be related.